tests/unit/OptionsListUtilsTest.ts

```diff
@@ -675,13 +675,13 @@ describe('OptionsListUtils', () => {
         );
         expect(options.personalDetails).toEqual(expect.arrayContaining([expect.objectContaining({login: CONST.EMAIL.MANAGER_MCTEST})]));
 
-        // Manager McTest should be included to recipients when the user has already submitted an expense
```

Comment:
This comment is still valid, we don't show this to users who dismiss the tooltip || who have already submitted an expense